privacy-on-beam/pbeam/count_test.go

```diff
@@ -230,6 +264,38 @@ func TestCountCrossPartitionContributionBounding(t *testing.T) {
 	}
 }
 
+// Checks that Count with partitions bounds per-user contributions correctly.
+func TestCountWithPartitionsCrossPartitionContributionBounding(t *testing.T) {
+	// pairs contains {1,0}, {2,0}, …, {50,0}, {1,1}, …, {50,1}, {1,2}, …, {50,9}.
+	var pairs []pairII
+	for i := 0; i < 10; i++ {
+		pairs = append(pairs, makePairsWithFixedV(50, i)...)
+	}
+	result := []testInt64Metric{
+		{0, 150},
+	}
+	p, s, col, want := ptest.CreateList2(pairs, result)
+	col = beam.ParDo(s, pairToKV, col)
+
+	partitions := []int{0, 1, 2}
```

Comment:
We are not actually testing cross-partition contribution bounding if we have a total of 3 kept partitions. We should keep more partitions and let Privacy on Beam drop the extra ones. So, let's say you keep `{0, 1, 2, 3, 4}`.

Reply:
I am... Since each of 50 users contributes to three partitions, I am making sure that the partitions that are contributed to are those three partitions that I keep. I am testing that the bounds are not wasted on unspecified partitions.